### ISSUE TITLE
update release (esr) start page

### ISSUE THREAD

**Issue by wsmwk** [thundernest/thunderbird-website#309]
Based on the excellent work in #301, we can proceed to update the release start page https://start.thunderbird.net/en-US/release/

[Many of the beta start page goals](https://github.com/thundernest/thunderbird-website/issues/301#issuecomment-1115977938) apply to the release start page, for example less words, more clarity.  (the current release page isn't as bad as the old beta page in terms of density, **except** donations.  

@ryanleesipes, @Alecaddd  On that subject, donations could use rethink. Paraphrasing from #302, how do we address complaints from users about donations?  We see these complaints frequently where the user states "how do I turn off the start page because ...
a) “every day you pester me about donations” (regardless of whether I have donated),
b) “I have already donated, but you ask anyway - don’t you know that I have already donated?”

In short, how do we acknowledge/thank/tip of the hat to those who have already donated?  And use text that makes it less likely the user will want to turn off the start page.

Also suggestions can now be directed to https://connect.mozilla.org/

Other concerns, goals, things to address before I start a document for the language?

Lastly, it would be nice to have this ready in two weeks for when version 102 becomes beta, in case we need to use it to solicit for more beta users, which we dearly want for QA of v102.

**Comment by wsmwk** [comment]
@Alecaddd,

Apparently it is taking a long time to getting localized versions of the beta start page, so we will want to finish this ticket asap.
 
There's almost nothing I would reword from the beta page. And I don't see anything that must brought from the "old" esr page.  So just minor adjustments mentioned above and (removing) beta stuff that obviously doesn't apply and no text document to offer.

Couple ideas ...

Perhaps repurpose "Learn and Connect" to 1) point to support KB link  2) also point to connect where they can offer both praise and ideas/suggestions

Under "Help" maybe add "Consider using Beta to give us feedback" and point to https://www.thunderbird.net/download/beta/  (we could really use more exposure)

For donations, were you thinking just keep the logo material in the top right and omit the text of the old page ("Why we need donations" and "Donate to Thunderbird"?  If you do plan to keep some text, can we add a thanks/shout out to people who **have** donated?   Or, perhaps just incorporate in the opening "Thank you for using Thunderbird and thank you for donating. Your experiences with Thunderbird are important, so please report ..."  (kinda kills two birds with one stone - thanks those who have donated, and may encourage someone who hasn't to do so)

Only other thought is pointing out the ability to customizing Thunderbird through add-ons and settings, and maybe pick a couple items from the Help menu?  For example Help > Tour points to https://www.thunderbird.net/en-US/features/   (OTOH perhaps not very useful if you have plans to raise visibility of features and customization in other ways)
